rero_invenio_base/cli/es/task.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+#
+# RERO Invenio Base
+# Copyright (C) 2023 RERO.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click elasticsearch tasks command-line utilities."""
+import sys
+from pprint import pformat
+from time import sleep
+
+import click
+from invenio_search import current_search_client
+from invenio_search.cli import es_version_check, with_appcontext
+
+
+def abort_if_false(ctx, param, value):
+    """Abort command is value is False."""
+    if not value:
```

Comment:
if the purpose is validate than `value === False`, you need to write `if value is False:` because `0`, `[]`, `{}`, ... are also evaluated to `False`

Reply:
This is the offical code from invenio.